src/getDescription.js

```diff
@@ -0,0 +1,31 @@
+export function getDescription() {
+  const metaDescription = document.querySelector(`meta[name="description"]`);
+  const metaOgDescription = document.querySelector(
```

Comment:
변수명 중점적으로 보는 것을 원하신다고 하셔서, Element인지 string인지 구분하기 위해 다음과 같은 것은 어떨까요?
근데 이건 취향차인가 싶기도 합니다:)

```suggestion
  const descriptionMetaElement = document.querySelector(`meta[name="description"]`);
  const ogDescriptionMetaElement = document.querySelector(
```

Reply:
변수명 지정에 많은 고민을 하고 있었는데 제안 주신 변수명으로 업데이트 완료했습니다:) 꼼꼼하게 봐주셔서 감사합니다!